tests/conftest.py

```diff
@@ -34,19 +34,20 @@ def migrate_test_db() -> None:
     logger.debug("Migrations successfully applied")
 
 
-@pytest.fixture(autouse=True, scope="session")
-def set_os_env() -> Generator:
-    """Sets an environment variable to tell the application it is in test mode."""
-    os.environ["TESTING"] = "True"
-    yield
-    os.environ["TESTING"] = "False"
+# @pytest.fixture(autouse=True, scope="session")
+# def set_os_env() -> Generator:
+#     """Sets an environment variable to tell the application it is in test mode."""
+#     os.environ["TESTING"] = "True"
+#     yield
+#     os.environ["TESTING"] = "False"
```

Comment:
This was the fix for loading in the correct test database — let's not leave this commented out.